modules/oe_translation_poetry/README.md

```diff
@@ -2,14 +2,61 @@
 
 This submodule integrates with the DGT Poetry system to provide translations services.
 
-For using the Mock, enable the OE Translation Poetry Mock submodule and add the following to your settings file:
+## Requirements
+
+If you want to use this submodule, you need to add the following packages to your site:
+
+* ec-europa/oe-poetry-client:dev-master
+
+## Setup
+
+### DGT request
+
+To integrate with Poetry, a request has to made to register the site with DGT. See [here](https://github.com/ec-europa/platform-dev/tree/release-2.5/profiles/common/modules/features/nexteuropa_dgt_connector/#requesting-access-to-the-dgt-connector) more information about this. Note: This documentation needs an update.
+
+### Configuration
+
+There are a number of required configuration options needed for the Poetry integration to work. These can be added at this path: `admin/tmgmt/translators/manage/poetry`:
+
+* Poetry service URL - the Poetry endpoint. For the acceptance environment this is `http://intragate.test.ec.europa.eu/DGT/poetry_services/components/poetry.cfc?wsdl`
```

Comment:
Also production URL should be provided.

Reply:
what is the prod URL?

Reply:
I remove also the test ones. We should not provide these publicly